sigstore/_internal/trust.py

```diff
@@ -214,66 +217,92 @@ def __str__(self) -> str:
         return self.value
 
 
-class TrustedRoot(_TrustedRoot):
-    """Complete set of trusted entities for a Sigstore client"""
+class TrustedRoot:
+    """
+    The cryptographic root(s) of trust for a Sigstore instance.
+    """
 
-    purpose: KeyringPurpose
+    class TrustedRootType(str, Enum):
+        """
+        Known Sigstore trusted root media types.
+        """
+
+        TRUSTED_ROOT_0_1 = "application/vnd.dev.sigstore.trustedroot+json;version=0.1"
+
+        def __str__(self) -> str:
+            """Returns the variant's string value."""
+            return self.value
+
+    def __init__(self, inner: _TrustedRoot):
+        """
+        Construct a new `TrustedRoot`.
+
+        @api private
```

Comment:
This is a `pdoc` (our document generator) thing -- it marks an otherwise public API (like the ctor) as not receiving public documentation, so that we don't commit to 3p proto types in our public APIs 🙂 

(Then again, it's entirely redundant here since `_internal.trust` is already undocumented as a completely private API.)